src/client/components/ReviewAndSubmitSection.jsx

```diff
@@ -47,81 +47,97 @@ class ReviewAndSubmitSection extends React.Component {
 
         {/* TODO(crew): Change names of sections to real names. */}
         <ReviewCollapsiblePanel
+            additionalClass="confirm-personal-information"
```

Comment:
nit: please prefix css classes that are used for non-style reasons by JS with `js-`